### ISSUE TITLE
Building latest 1.10.x from command line

### ISSUE THREAD

**Issue by dmhiggins23** [OrchardCMS/Orchard#8758]
I'm just updating my orchard application. It compiles in Visual Studio 2022, but does not compiled using the command line build.

It give a syntax error message "; expected" for the following line

https://github.com/OrchardCMS/Orchard/blob/90b104ed74234bdaaedf2b730166eddc66ad4d4c/src/Orchard.Web/Modules/Orchard.Email/Services/SmtpMessageChannel.cs#L157

I believe that there is a version conflict between the compiler that is being called from the command line and the version of the compiler required to handle the c# language constructs used in SmtpMessageChannel.cs (version 8, I believe). I believe that ClickToBuild.cmd determines which version of msbuild to use, and in my case it is using a version that does not support the more advanced language feature. 

Any suggestions on whether this is due to version issues on my computer or if the ClickToBuild.cmd file can be updated to resolve the issue?

Thank you!

**Comment by sebastienros** [comment]
Looks like we need to update this line https://github.com/OrchardCMS/Orchard/blob/dev/ClickToBuild.cmd#L6 to search within 19.0 too. And the lines after to take the result into account. Can you try it and make a PR if that works?

**Comment by dmhiggins23** [comment]
I've create a pull request. Since VS2017 (MSBuild v15) and earlier can no longer compile the newer c# language features used in SmtpMessageChannel.cs, I removed these as build alternatives. I also made a change to the file  SmtpSettingsPart.cs so that VS2019 (MSBuild v16) could still compile orchard. In the process, I noticed that the version of nuget.exe no longer worked, so I updated it to the latest from https://www.nuget.org/downloads (v 6.9.1).
